src/state/farms/hooks.ts

```diff
@@ -166,9 +166,11 @@ export const useFarmsData = (isIncludeOutsideFarms = true) => {
 
       const farmsData = await getBulkPoolDataFromPoolList(poolAddresses, apolloClient, ethPrice.currentPrice, chainId)
 
-      const rewardTokens = rewardTokenAddresses.map(address =>
-        address.toLowerCase() === ZERO_ADDRESS.toLowerCase() ? nativeOnChain(chainId as ChainId) : allTokens[address],
-      )
+      const rewardTokens = rewardTokenAddresses
```

Comment:
ké: fix that bug
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/33005392/190917608-1513e900-d110-4eaa-b725-5b6d3e5664af.png">